lib/yard/registry_store.rb

```diff
@@ -291,7 +291,8 @@ def load_object_types
     def load_checksums
       return unless File.file?(checksums_path)
       lines = File.readlines(checksums_path).map do |line|
-        line.strip.split(/\s+/)
+        parts = line.strip.split(/\s+/)
+        parts.count > 2 ? [parts[0..-2].join(' '), parts[-1]] : parts
```

Comment:
Might be simpler to use:

```suggestion
line.strip.rpartition(' ').tap {|p| p.delete_at(1) }
```

(replaces the first line too, GitHub doesn't support multiline suggestions yet)

Reply:
I like this approach better too, thanks for pointing it out. 👍